app/javascript/crayons/CTAs/__stories__/CTAs.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import { h } from 'preact';
+import { CTA } from '..';
+import SampleIcon from '../../../../assets/images/cog.svg';
+
+export default {
+  component: CTA,
+  title: 'Components/BETA/CTAs',
+  argTypes: {
+    variant: {
+      options: ['default', 'branded', 'ghost'],
+      control: 'select'
+    },
+  },
+};
+
+export const Default = (args) => <CTA {...args} />;
+Default.args = {
+  variant: 'default',
+  href: '#',
+  rounded: false,
+  children: 'Call To Action',
+};
+
+export const Branded = (args) => <CTA {...args} />;
+Branded.args = {
+  ...Default.args,
+  variant: 'branded',
+};
+
+export const Ghost = (args) => <CTA {...args} />;
+Ghost.args = {
+  ...Default.args,
+  variant: 'ghost',
+};
```

Comment:
I'm still a little bit on a fence whether or not we need that variant. I was primarily thinking about all the places where we use block links (left sidebar with main nav, all sorts of dropdowns, tabs, etc.) - these should all share the same basic style. But on the other hand I'm not sure if these should be called "CTAs" or just "Links" or "Block Links" or "Nav Links"...

Reply:
Yeah I kinda think they're not "CTAs", but something like "Block Link" makes sense to me.